config/sources/families/bcm2711.conf

```diff
@@ -119,7 +114,7 @@ function post_family_tweaks_bsp__add_hooks_to_move_kernel_initrd_and_dtb() {
 		target=/boot/firmware
 		dtb_dir=/usr/lib/linux-image-$abi/
 
-        # The kernel image can be named either "Image" (standardized for ARM platforms) 
+        # The kernel image can be named either "Image" (standardized for ARM platforms)
         #   or "vmlinuz-${abi}" (default naming convention for Linux kernels). We need
 		#   too support both conventions.
```

Comment:
nitpick "to support" not "too support"

can you add a commit to this effect if this is still an active PR?